test/framework/fileConversion.go

```diff
@@ -49,29 +49,27 @@ func tarCmd(src, tgtDir string) (string, error) {
 }
 
 func gzCmd(src, tgtDir string) (string, error) {
```

Comment:
There's a gzip/xz flag to keep the file, however that doesn't help us when only running a gzip test since it will leave a 2nd .iso laying around.

Without the flag though, a .iso that another test might be expecting will be destroyed.